CHANGELOG.md

```diff
@@ -46,12 +80,16 @@
 
 **Closed issues:**
 
+- Cannot use custom Windows AMI  [\#381](https://github.com/chef/chef-provisioning-aws/issues/381)
```

Comment:
So most of the time when it adds old notes from this it means you need to specify `--exclude-labels "Exclude From Changelog"` - thats the case with this issue